tests/e2e/setup/utils.go

```diff
@@ -116,33 +121,36 @@ func (s *E2ETestSuite) SendIBC(srcChainID, dstChainID, recipient string, token s
 			"failed to send IBC tokens; stdout: %s, stderr: %s", outBuf.String(), errBuf.String(),
 		)
 
-		// don't check for the tx hash if we expect this to fail due to quota
-		if strings.Contains(errBuf.String(), "quota transfer exceeded") {
-			s.Require().True(failDueToQuota)
-			return
-		}
-
-		re := regexp.MustCompile(`[0-9A-Fa-f]{64}`)
-		txHash := re.FindString(errBuf.String() + outBuf.String())
-
-		// retry if we didn't get a txHash
-		if len(txHash) == 0 && i < 4 {
-			continue
-		}
-
-		s.Require().NotEmptyf(txHash, "failed to find transaction hash in output outBuf: %s  errBuf: %s", outBuf.String(), errBuf.String())
-		endpoint := s.UmeeREST()
-		if strings.Contains(srcChainID, "gaia") {
-			endpoint = s.GaiaREST()
-		}
-
-		s.Require().Eventually(func() bool {
-			err := s.QueryUmeeTx(endpoint, txHash)
-			if err != nil {
-				s.T().Log("Tx Query Error", err)
-			}
-			return err == nil
-		}, 5*time.Second, 200*time.Millisecond, "require tx to be included in block")
+		// Note: we are cchecking only one side of ibc , we don't know whethever ibc transfer is succeed on one side
```

Comment:
I am commenting this LOC, becuase we are checking only one side of ibc txn , we don't know whether  ibc is succeed on other side of ibc

Reply:
was this working before? Why it doesn't work now?

Reply:
No, it is not working previously because they check only one side on ibc

Reply:
Previously When I wrote this , there is no txn checking , we are just waiting around 12 seconds I think for every IBC Txn